src/components/@core/navigation-menu/light/left-menu/logo.tsx

```diff
@@ -23,15 +26,21 @@ const Logo = styled.div`
     }
   }
 
-  button {
+  button.ibpmenu {
     display: none;
-    padding: 0.5rem;
+    padding: 0.5rem 1rem;
     font-size: 1.5rem;
   }
 
   ${Mq.max.sm} {
     width: 100%;
-    button {
+    a {
+      margin: 0px 5px;
+      img {
+        width: 4rem;
+      }
+    }
```

Comment:
what this code does as there no links added?

Reply:
@harshzalavadiya , the image and group name is wrapped with link, so 
![Screenshot from 2020-07-02 19-46-33](https://user-images.githubusercontent.com/30981535/86370799-c9d09980-bc9d-11ea-9179-0cbf6d2f95af.png)
![Screenshot from 2020-07-02 19-46-29](https://user-images.githubusercontent.com/30981535/86370805-cc32f380-bc9d-11ea-8634-238f898be73e.png)

as we get to sx the image size needs to be reduced.
The margin will be removed anyways

Reply:
@teamdraftbox 

This width was given fo logos that are bigger in width for example the western ghats logo, assam biodiversity logo etc.

So to conclude if we reduce width that will make logo too small so we have to find alternative to this